deploy-continuous-integration.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -xe
```

Comment:
is the print flag intentional here?

Reply:
Not sure! I copied/pasted [from the p5-replay project](https://github.com/code-dot-org/p5-replay/blob/d26c5b120cf1799a089237918045c3ade794945b/deploy-ci.sh#L1).

Reply:
Hah, gotcha. I think it makes sense to keep around in a script file that's only doing a few things, but note this flag can quickly become the source of a whole lot of noise as scripts grow